draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -788,33 +768,53 @@ Content-Type: text/plain
    <iref item="URI scheme" subitem="https"/>
 <t>
    The "https" URI scheme is hereby defined for the purpose of minting
-   identifiers according to their association with the hierarchical
-   namespace governed by a potential HTTP origin server listening to a
-   given TCP port for TLS-secured connections (<xref target="RFC8446"/>).
-</t>
-<t>
-   All of the requirements listed above for the "http" scheme are also
-   requirements for the "https" scheme, except that TCP port 443 is the
-   default if the port subcomponent is empty or not given,
-   and the user agent &MUST; ensure that its connection to the origin
-   server is secured through the use of strong encryption, end-to-end,
-   prior to sending the first HTTP request.
+   identifiers within a hierarchical namespace governed by a potential origin
+   server capable of establishing a secure and verifiable TLS connection on
```

Comment:
I think that we need to be more precise about "secure and verifiable" here.  Using terms of art, you would say "a TLS connection that is authenticated on behalf of the identified authority and provides confidentiality and integrity protection (`<xref...`)."

Reply:
done